include/boost/unordered/unordered_flat_map.hpp

```diff
@@ -202,6 +203,13 @@ namespace boost {
         return *this;
       }
 
+      unordered_flat_map& operator=(std::initializer_list<value_type> il)
+      {
+        clear();
+        insert(il);
```

Comment:
You could save a function call by instead writing
```cpp
insert(il.begin(), il.end());
```

Reply:
Done, thank you!